src/viewtransaction/viewTransaction.es6

```diff
@@ -233,11 +239,11 @@ const updateIndicative = (data, state) => {
     state.proposal_open_contract.current_spot = contract.current_spot;
     state.proposal_open_contract.current_spot_time = contract.current_spot_time;
     state.proposal_open_contract.bid_price = contract.bid_price;
-    state.proposal_open_contract.entry_tick = contract.entry_tick;
+    state.proposal_open_contract.entry_tick = countDecimals(contract.entry_tick) < 3 ? contract.entry_tick.toFixed(3) : contract.entry_tick;
```

Comment:
Why is this necessary?

Reply:
It's because the entry/exit spot doesn't show enough decimal places. For example `123.45` is supposed to be shown `123.450`. Sometimes they choose a `match/differ` contract and last digit prediction as 5, which means they lost in this case, but they see last digit as 5. They will get really angry and complain 😂

Reply:
👍 ah yes, I think we can use `display_decimals` instead of `3`?

Reply:
Oh okay, I'll change it